src/api/.rubocop.yml

```diff
@@ -141,6 +141,9 @@ Style/SpaceInsideBrackets:
 Style/TernaryParentheses:
   Enabled: true
 
+# Checks for receiver.length == 0 predicates and the negated versions receiver.length > 0 and receiver.length != 0.
```

Comment:
what about _Checks for predicates than can be replaced by receiver.empty? and !receiver.empty?_ ?